Common/src/CConfig.cpp

```diff
@@ -4856,6 +4861,10 @@ void CConfig::SetPostprocessing(unsigned short val_software, unsigned short val_
     }
 #endif
 
+    /*--- Use the same linear solver on the primal as the one used in the adjoint. ---*/
+    Kind_Linear_Solver = Kind_DiscAdj_Linear_Solver;
+    Kind_Linear_Solver_Prec = Kind_DiscAdj_Linear_Prec;
```

Comment:
Here again that would be nice to have documented in the config_template or CConfig.

Tbh I wasn't aware that so far I have always used FGMRES+ILU in DA computations by not setting `DISCADJ_LIN_*`. I thought that the `LINEAR_SOLVER` stuff is used for the evalutaion of the linear solver during tape evaluation as well. And `grep`pring the Testcase folder showed me that only fea or fsi cases used this option.  Or am I missing something here?

But I see your point pedro in #942 and together with the `MATH_PROBLEM` which can be deletd from the `.cfg` these cases one can use the exact same config which is nice. I guess it doesn't make too much sense to use a different Linsol configuration during the taping compared to tape evaluation ... but I might ask about that on wednesday during the dev-meeting again :)